### DISCUSSION TITLE
Mirror containers to Github Container Registry

### DISCUSSION THREAD

**Discussion by tomschlick** [serversideup/docker-php#218] [Ideas & Feature Requests]
Now that you're working on the new version 3 beta, it might be a good time to start mirroring container images to the [Github Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry). 

While the images would be exactly the same as those pushed to DockerHub, some benifits include faster push/pull times for those of us using Github Actions, direct integration with the repo to show the containers, and additional redundancy in case DcokerHub [tightens the belt like they started to](https://www.docker.com/pricing/october-2022-pricing-change-faq/) with private repos last year.

**Comment by jaydrogers** [comment]
Funny that you posted this, because I was thinking of doing this the other day.

Adding some notes for myself:

### Example of publishing to Dockerhub and GHCR
https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-docker-hub-and-github-packages

### Build & Push in a single action
https://github.com/marketplace/actions/docker-build-push-action#auto-tagging-with-gitops

**Comment by jaydrogers** [comment]
Closing this discussion since it has been developed and is being tracked in the issue below 🥳 

https://github.com/serversideup/docker-php/issues/231#issuecomment-1832785767
